src/main/java/me/botsko/prism/Prism.java

```diff
@@ -326,7 +327,16 @@ public void onEnable() {
         } else {
             pasteKey = null;
         }
+        final List<String> worldNames = getServer().getWorlds().stream()
+                .map(World::getName).collect(Collectors.toList());
+        final String[] playerNames = Bukkit.getServer().getOnlinePlayers().stream()
+                .map(Player::getName).toArray(String[]::new);
         // init db async then call back to complete enable.
+        final BukkitTask updating = Bukkit.getScheduler().runTaskTimerAsynchronously(instance, () -> {
+            if (!isEnabled()) {
+                warn("Prism is loading and updating the database logging is NOT enabled");
```

Comment:
I think a semicolon is needed here:
`Prism is loading and updating the database; logging is NOT enabled`

Reply:
I would add some blank lines before these variable declarations, but that's up to you (`worldNames`, `playerNames`, `updating`)